File: Domain/Models/RevisionTecnica/RevisionTecnica.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class RevisionTecnica : ClaseBase
    {
        [Key]
        public int RevisionTecnicaId { get; set; }
        List<RevisionTecnicaConcepto> Conceptos { get; set; }
    }
}

File: CarAgency/ClientApp/src/models/RevisionTecnica.ts
import RevisionTecnicaConcepto from "models/RevisionTecnicaConcepto";

export default class RevisionTecnica {
    RevisionTecnicaId?: number;
    Conceptos?: RevisionTecnicaConcepto[];

    public constructor(){
        this.RevisionTecnicaId = undefined;
        this.Conceptos = undefined;
    }



}